src/client/LiveStream.js

```diff
@@ -799,86 +797,81 @@ exports.StreamStopSession = async function({name}) {
 
     this.SetNodes({fabricURIs: [fabURI]});
 
-    let edgeWriteToken = mainMeta.live_recording.fabric_config.edge_write_token;
+    const metaEdgeWriteToken = mainMeta.live_recording.fabric_config.edge_write_token;
 
-    if(edgeWriteToken === undefined || edgeWriteToken === "") {
+    if(metaEdgeWriteToken === undefined || metaEdgeWriteToken === "") {
```

Comment:
!metaEdgeWriteToken? or does it matter if it's null

Reply:
Changing

Reply:
Ok changed this